src/main/target/STM32_UNIFIED/target.h

```diff
@@ -263,6 +263,8 @@
 #define USE_BARO_SPI_QMP6988
 #define USE_BARO_DPS310
 #define USE_BARO_SPI_DPS310
+#define USE_BARO_2SMBP_02B
+#define USE_BARO_SPI_2SMBP_02B
```

Comment:
Should go somewhere here now:
https://github.com/betaflight/betaflight/blob/28c3f4c7e4e2f470daad64c309c2d7054ffb2e84/src/main/target/common_post.h#L47-L61